index.js

```diff
@@ -94,6 +95,8 @@ module.exports = function(connect) {
 			}
 			return store.knex(store.tablename).del()
 			.whereRaw(condition, dateAsISO(store.knex));
+		}).then(function() {
```

Comment:
You're probably correct. I guess the cleanup failing here is an operational error that is fairly safe just to retry on the interval. I'll submit another PR with the requested change.